### DISCUSSION TITLE
Where is InitPropVariantFromString?

### DISCUSSION THREAD

**Discussion by optsing** [microsoft/CsWin32#1234] [Q&A]
If I add `InitPropVariantFromString` to `NativeMethods.txt`, I get the warning message 

> Method, type or constant "InitPropVariantFromString" not found. Did you mean  or "InitPropVariantFromStringVector" or "InitPropVariantFromStringAsVector"?
 
Why is this method missing? What is the correct way to create PROPVARIANT without it?

**Comment by AArnott** [comment]
It's missing from the metadata, but [it is a documented function](https://learn.microsoft.com/en-us/windows/win32/api/propvarutil/nf-propvarutil-initpropvariantfromstring). I've created an issue in the metadata repo: https://github.com/microsoft/win32metadata/issues/1946

**Comment by catmanjan** [comment]
So, what is the correct to create a PROPVARIANT?
